roles/cnf_cert/tasks/logging.yml

```diff
@@ -67,8 +67,15 @@
 # this feature is enabled (disabled by default).
 # This check can only be done if CNF Cert was executed correctly (so claim.json file exists).
 - name: Confirm we are testing the correct CNF Cert code
+  vars:
+    log_file_name: >
+      {% if test_network_function_version is version("v5.1.0", "<=") and test_network_function_version != "HEAD" %}
+        dci-tnf-execution.log
+      {% else %}
+        certsuite.log
+      {% endif %}
   ansible.builtin.lineinfile:
-    path: "{{ tnf_logs_location }}/dci-tnf-execution.log"
+    path: "{{ tnf_logs_location }}/{{ log_file_name | trim }}"
```

Comment:
LGTM, just to confirm. looks like in TNF v5.1.1 we only have `certsuite.log` log right? in the job with v5.1.0 version I see both, the `certsuite.log` and `dci-tnf-execution.log`

Reply:
Yes, it was renamed at some point to `certsuite.log`, so, now that for v5.1.1 I'm proposing to directly use the container with `podman run`, then I've omitted the capture of logs in the file called `dci-tnf-execution.log`, that's why this file is not generated in 5.1.1, and with `certsuite.log`, it should be enough to check what we need to check in this task.
Also, there are some conditionals in the code to ensure backwards compatibility. When all consumers of this role move to, at least, 5.1.1, I'll start removing the support of the old code, as I did some months ago with 4.x.x.

Reply:
I've finally kept dci-tnf-execution.log file for tnf v5.1.1 too, so I've removed this new code and I'll keep using the old one.